Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    jekyll-crds (1.2.3)
+    jekyll-crds (1.3.0)
```

Comment:
Isn't this a patch since we aren't changing any existing functionality?

I'd propose 1.2.4, just adding as a comment because...meh

Reply:
This PR adds a new filter so it isn't really existing functionality.